pkg/stores.go

```diff
@@ -1,41 +1,76 @@
 package ocr2keepers
 
-import "fmt"
+// NotifyOp is an operation that can be notified by the ResultStore
+type NotifyOp uint8
 
-type ResultStore[T any] interface {
-	Add(T)
-	Remove(T)
-	View() ([]T, error)
+const (
+	NotifyOpNil NotifyOp = iota
```

Comment:
This isn't about your PR, but I just wanted to say that I hate iotas 😂 I think they're a pain to debug with! But I get that we use them in lots of places in chainlink so I think it's fine to keep this as-is! 👍